nimare/meta/mkda.py

```diff
@@ -725,6 +724,12 @@ class KDA(CBMAEstimator):
 
     Available correction methods: :func:`KDA.correct_fwe_montecarlo`
 
+    Warnings
+    --------
+    The KDA algorithm has been replaced in the literature with the MKDA algorithm.
+    As such, this estimator should almost never be used, outside of systematic
+    comparisons between algorithms.
```

Comment:
This may help dissuade people from using the KDA estimator.

Reply:
Well, we can hope. :)

Reply:
To be honest, I doubt anyone will use anything except ALE (and maybe MKDAChi2, out of familiarity with Neurosynth), given that Salimi-Khorshidi paper. Unless we get SDM working, of course.

Reply:
I'm not sure about that; there are clear principled trade-offs between ALE and MKDA. The former is too liberal, the latter is too conservative; the ALE kernel is more biologically plausible, but requires stronger assumptions and produces less interpretable values. With KDA, it seems harder to argue that any gains justify the fundamentally bad fixed effect assumption.

If you mean that people are just going to use whichever method they think gives "better" results (in the "more likely to produce significance" sense), there's probably something to that, but that's potentially also something we could mitigate to some degree with good best practices documentation.

It does occur to me that the flaws in KDA could in principle be addressed without going full MKDA—e.g., by allowing non-binary values at each voxel, but weighting the overall map in proportion to the MKDA CIM (i.e., overlapping peaks would produce values > 1, but the overall mean across all values with non-zero activation would be 1). This would in a sense be something intermediate between MKDA and ALE, both in terms of sensitivity/specificity profile and in terms of strength of assumptions. Once we have a good performance test suite, it would be pretty trivial to implement that and test it out if we were so inclined.

Reply:
I meant more that people will just use the algorithm for which they have a clean citation. Salimi-Khorshidi found, in a specific situation, that ALE outperformed MKDA, and that eight year old conclusion is going to be the end of the investigation for most folks. At least until we publish a comparison study of our own.